src/app/icd/client/ICDClientStorage.h

```diff
@@ -76,8 +79,12 @@ class ICDClientStorage
      * and populate the clientInfo with the stored information corresponding to the key.
      * @param[in] payload received checkIn Message payload
      * @param[out] clientInfo retrieved matched clientInfo from storage
+     * @param[out] counter counter value received in the CheckIn message
```

Comment:
Again, is it "CheckIn" or "check-in" or something else?  Please be consistent.  The comments on the PR said this had been resolved, but apparently not...

Reply:
Updated, please re-review.